app/models/kit.rb

```diff
@@ -30,6 +30,7 @@ class Kit < ApplicationRecord
   validates :name, uniqueness: { scope: :organization }
 
   validate :at_least_one_item
+  validate :line_items_quantity_is_positive
```

Comment:
You don't need to add a new method, you can use a block:

`validate {  :line_items_quantity_is_at_least(1) }`